src/flows/components/panel/FlowPanel.tsx

```diff
@@ -165,6 +183,15 @@ const FlowPanel: FC<Props> = ({
                     className="flows-config-panel-button"
                   />
                 </FeatureFlag>
+                {isVisible &&
```

Comment:
kept the previous `isVisible` flag here

Reply:
Right, because this shouldn't be accessible when the panel is hidden